Bootcamp/Lecture4/Homework/KFedakTask4/KFedakTask4/PhoneEmergencyTestHolder.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KFedakTask4
+{
+    class PhoneEmergencyTestHolder : IEnumerable
+    {
+        public List<ITelephone> telephones = new List<ITelephone>();
+
+        public PhoneEmergencyTestHolder()
+        {
+            telephones.Add(new IPhone(4, "Iphone13"));
+            telephones.Add(new Nokia(8, "345"));
+            telephones.Add(new IPhone(25, "Iphone7"));
+        }
+
+        public void TestEmergency()
```

Comment:
In the requirements `TestEmergency` should take `PhoneEmergencyTestHolder` as an argument
```suggestion
        public static void TestEmergency(PhoneEmergencyTestHolder holder)
```

Reply:
made